### DISCUSSION TITLE
Embarrassing! Private/public repo depending on branch?

### DISCUSSION THREAD

**Discussion by mktbup** [community/community#23684] [Other Feature Feedback, Questions, & Ideas]
<p>The boss wants me to write 2 versions of a book: 1 freely available and MIT licensed, then a branch that’s a paid version with additional material, where the additional material is proprietary. Can that be done?</p>

**Comment by Maximization** [comment, marked as answer]
<p>Hi! You can only restrict push access to a branch, but not its visibility. I recommend using two repositories: a public repository for the MIT book and a private repository for the proprietary material.</p>
